store/store_action.go

```diff
@@ -289,6 +296,10 @@ func genInstanceKey(curSnap *CurrentSnap, salt string) (string, error) {
 	return fmt.Sprintf("%s:%s", curSnap.SnapID, enc), nil
 }
 
+func minStoreVersion(storeVer *int, minVer int) bool {
```

Comment:
If we end up keeping this, calling this `meetsVersion` (or something like that) would be more self-explanatory IMO

Reply:
I removed it completely, I totally agree on making the store version check explicit (e.g. `(storeVer == nil || *storeVer >= 54)`).